package.json

```
@@ -12,6 +12,8 @@
     },
```

Best practice to provide engines. Which will give an overview of the stacks that used in project.
```suggestion
    },
    "engines": {
    "node": "16.x",
    "npm": "8.19.x",
    "yarn": "1.22.x"
  },
```